packages/yoastseo/src/scoring/ASSESSORS OVERVIEW.md

```diff
@@ -1,8 +1,10 @@
-# SEO Assessors
-## Which assessor?
-[Overview of used SEO assessors](https://drive.google.com/file/d/1jOEpNK0ZcyCAKpjaiJuBywuqNC08tlW5/view?usp=sharing) in the `yoastseo` package
+# Assessor types
+## SEO Assessors
+### Which assessor?
+Overview of the used SEO assessors in the `yoastseo` package:
 
-## Standard SEO assessor (Focus keyphrase)
+![Screenshot of the overview of the SEO assessors](/packages/yoastseo/images/assessorsOverview.png)
```

Comment:
I add the image directly in the repo to avoid broken link. In our google drive, this image has been moved but the link here wasn't adjusted.